themes/sbvfrd/js/swissbib/Accordion.js

```diff
@@ -0,0 +1,110 @@
+/**
+ * Manage accordion auto expand/collapse actions on record holding tab
+ *
+ */
+swissbib.Accordion = {
+
+  /**
+   * @var {String}
+   */
+  cookieName: 'expandedGroups',
+
+  /**
+   * @var {Number}
+   */
+  idRecord: null,
+
+  /**
+   * Initialize for record
+   *
+   * @param  {Number}  idRecord
+   */
+  initRecord: function(idRecord) {
+      this.idRecord = idRecord;
+  },
+
+  /**
+   * Saves state of expanded groups
+   *
+   */
+  saveExpandedGroups: function() {
+      var expandedGroupIds = [];
+
+      $("#accordion .in").each(function (index) {
+          var currentItem = $(this);
+
+          //only saves state of first hierachie level
+          if (currentItem.attr('level') == 1) {
+              expandedGroupIds.push(currentItem.attr('id'));
+          }
+      });
+
+      if(expandedGroupIds.length > 0) {
+          var cookieData = {};
```

Comment:
IMHO there's no need to save data by idRecord if you create a new object every time...